projects/observability/src/shared/graphql/request/builders/specification/explore/explore-specification-builder.ts

```diff
@@ -1,14 +1,33 @@
+import { DateCoercer } from '@hypertrace/common';
 import {
+  AttributeMetadataType,
   convertToGraphQlMetricAggregationType,
   GraphQlMetricAggregationType,
   MetricAggregationType
 } from '@hypertrace/distributed-tracing';
 import { GraphQlEnumArgument } from '@hypertrace/graphql-client';
+import { INTERVAL_START_QUERY_KEY } from '../../../../model/schema/explore';
 import { ExploreSpecification } from '../../../../model/schema/specifications/explore-specification';
 import { GraphQlObservabilityArgumentBuilder } from '../../argument/graphql-observability-argument-builder';
 
 export class ExploreSpecificationBuilder {
   private readonly argBuilder: GraphQlObservabilityArgumentBuilder = new GraphQlObservabilityArgumentBuilder();
+  private readonly dateCoercer: DateCoercer = new DateCoercer();
+
+  public exploreSpecificationForInterval(): ExploreSpecification {
+    return {
+      resultAlias: () => INTERVAL_START_QUERY_KEY,
+      name: INTERVAL_START_QUERY_KEY,
+      asGraphQlSelections: () => [],
+      extractFromServerData: serverData => ({
+        value: this.dateCoercer.coerce(serverData[INTERVAL_START_QUERY_KEY]),
+        type: AttributeMetadataType.Timestamp
+      }),
+      asGraphQlOrderByFragment: () => ({
```

Comment:
This will error right? should we just have it throw for now so we don't try to use it (or log and return no order by)

Reply:
okay. will check

Reply:
Will put up a separate PR(if required)